api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -38,7 +39,8 @@ public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements O
 	@Field
 	private String name;
 	
-	@Column(name = "description", length = 255)
+	@Column(name = "description", columnDefinition = "text")
+	@Lob
```

Comment:
I do not think a description of length 255 is a Lob.

Reply:
Some tables use text for the description field, while others use varchar. Defining description as text in the superclass and overriding it as varchar in the child classes when necessary helps mitigate compatibility issues between H2 and MySQL during tests

Reply:
Because the majority of the descriptions are small, it does not make sense to me making it a LOB in the base class.

Reply:
If we don't include lob here we'll get an error when running ValidHibernateMappings with H2.
`
Schema-validation: wrong column type encountered in column [description] in table [order_type]; found [clob (Types#CLOB)], but expecting [text (Types#VARCHAR)]`

if we set this to "varchar" and try to override to "text" when necessary we still get the error as we can't add @Lob for overrides.

Reply:
Thinking about this again, which object would need text that is too big to require a LOB, for just a mere description?

Reply:
Tables like global_property, order_type, and form use `text` for `description`.
Should we move the tables that use `text` for description to something like `varchar(1024)`?

Reply:
I think we should do so. I cannot think of a reason why any of those would require a LOB. Let us change them.